webcam-capture-drivers/driver-gst1/pom.xml

```diff
@@ -7,18 +7,19 @@
 		<artifactId>webcam-capture-drivers</artifactId>
 		<version>0.3.12-SNAPSHOT</version>
 	</parent>
-	
+
 	<artifactId>webcam-capture-driver-gst1</artifactId>
 	<packaging>jar</packaging>
-	
+
 	<name>Webcam Capture - GStreamer 1.x Driver</name>
 	<description>Webcam Capture driver using GStreamer 1.x framework to grab frames from camera devices</description>
 
 	<repositories>
 		<repository>
 			<id>ftp-repo</id>
 			<name>sarxos-ftp-repo</name>
-			<url>http://sarxos.pl/maven2</url>
+			<!-- <url>http://repo.sarxos.pl/maven2</url> -->
+			<url>http://repo.sarxos.pl/maven2/index.php</url>
```

Comment:
I think this should be without index.php. I'm not sure if this won't confuse Maven when it tries to download artifact from this repo.